velox/functions/prestosql/Slice.cpp

```diff
@@ -200,22 +203,35 @@ class SliceFunction : public exec::VectorFunction {
     return std::min(endIndex - start, length);
   }
 };
+} // namespace
+
+template <TypeKind Kind>
+void registerSliceFunction(const std::string& prefix) {
```

Comment:
I think it would be better to implement the register logic for presto/spark separately. For presto, only consider bigint. And for spark, only consider int.

Reply:
Do you mean we declare class `SliceFunction` in `Slice.h`, and implement different `registerSliceFunction` for spark/presto?